### PR TITLE
Add Golang Basics Part I (digits) task

### PR DESCRIPTION
**Pull request by kl177**

### CODE REVIEW

File: submissions/slirx/digits/main.go [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+		}
+
+		slice[k] = int(x)
+	}
+
+	return slice
+}
+
+// parse ASCII digits to a map of digits with a map of lines for each digit
+func parseDigits() map[int]map[int]string {
```
Comment (lempiy) [inline submissions/slirx/digits/main.go]: Think that `map[int]map[int]string` is probably  `[][]string` no ?

Comment (lempiy) [reply submissions/slirx/digits/main.go]: Or even
`[10][7]string`

File: submissions/slirx/digits/main.go [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+	// check symbol's length
+	if len(symbol) > 1 {
+		fmt.Println("ERROR: wrong symbol's length.")
+		os.Exit(1)
+	}
+
+	// check whether user passed argument
+	if len(args) == 0 {
+		fmt.Println("ERROR: can not find required argument.")
+		os.Exit(1)
```
Comment (lempiy) [inline submissions/slirx/digits/main.go]: You may probably use `log.Fatal` for that purpose.

File: submissions/slirx/digits/main.go [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+	numbers := make(map[int]map[int]string, 10)
+	lines := strings.Split(digits, "\n")
+	l := len(lines)
+	index := 0
+	line := 0
+
+	for i := 0; i < l; i++ {
+		// skip empty lines
+		if "" == lines[i] {
+			continue
```
Comment (lempiy) [inline submissions/slirx/digits/main.go]: Maybe, if you gonna split lines by `\n\n` not `\n`, you wont need that `continue` ?

Comment (lempiy) [reply submissions/slirx/digits/main.go]: I mean, split by `\n\n` at first and then by `\n`

Comment (lempiy) [reply submissions/slirx/digits/main.go]: Or you can split by regexp
https://golang.org/pkg/regexp/#Regexp.Split
`\n+`

**Review by lempiy — CHANGES_REQUESTED** [review]
Look good. A couple of cosmetic suggestions bellow.

**Comment by kl177** [comment]
done. but instead of splitting by \n\n I've replaced it with \n

**Pull request closed without merging**